android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -749,13 +749,7 @@ protected void evaluateJavascriptWithFallback(String script) {
         evaluateJavascript(script, null);
         return;
       }
-
-      try {
-        loadUrl("javascript:" + URLEncoder.encode(script, "UTF-8"));
-      } catch (UnsupportedEncodingException e) {
```

Comment:
Is it safe to remove this `try-catch`?

Reply:
Thanks for the review, I tried this:
```
try {
	loadUrl("javascript:(function() {\n" + script + ";\n})();");
} catch (UnsupportedEncodingException e) {
	// UTF-8 should always be supported
	throw new RuntimeException(e);
}
```
But it was always throwing an exception!

`try-catch` was added in these [changes ](https://github.com/facebook/react-native/commit/e6b305b722a8f52fad8ac8bf9f95204fa2192944), I don't think we need it anymore??